ttnctl/cmd/devices_disable.go

```diff
@@ -0,0 +1,125 @@
+// Copyright © 2017 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"strings"
+
+	"github.com/TheThingsNetwork/api"
+	"github.com/TheThingsNetwork/api/handler"
+	"github.com/TheThingsNetwork/api/handler/handlerclient"
+	ttnlog "github.com/TheThingsNetwork/go-utils/log"
+	"github.com/TheThingsNetwork/ttn/core/types"
+	"github.com/TheThingsNetwork/ttn/ttnctl/util"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func disableDevice(manager *handlerclient.ManagerClient, dev *handler.Device, flags *pflag.FlagSet) {
+	// Do all updates
+	if dev.GetLoRaWANDevice().AppKey.String() != "" {
+		old := dev.GetLoRaWANDevice().AppKey.String()
+		dev.GetLoRaWANDevice().AppKey = &types.AppKey{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01}
```

Comment:
Setting a key to something constant and known is a bad idea.

Reply:
Now setting empty AppKey